data/defscript/popups.kvs

```diff
@@ -18,17 +18,17 @@ defpopup(highlight)
 		}
 	}
 
-	item($tr("Stop Highlighting (Local)","defscript"),41) (%:beingHlTmp && !%:beingHl)
+	item($tr("Stop highlighting (Local)","defscript"),41) (%:beingHlTmp && !%:beingHl)
```

Comment:
I agree they all should be the same, but camel casing imo in this usage is more wrong than right.

Book titles, Headlines, blog posts titles, I can go with even if its debatable like you say, but here I disagree with the usage.